src/pixelator/cli/layout.py

```diff
@@ -0,0 +1,107 @@
+"""
+Console script for pixelator (layout)
+
+Copyright © 2024 Pixelgen Technologies AB.
+"""
+
+from pathlib import Path
+from typing import get_args
+
+import click
+
+from pixelator import read
+from pixelator.cli.common import logger, output_option
+from pixelator.graph.backends.protocol import SupportedLayoutAlgorithm
+from pixelator.pixeldataset.precomputed_layouts import (
+    generate_precomputed_layouts_for_components,
+)
+from pixelator.report.models.layout import LayoutSampleReport
+from pixelator.utils import (
+    create_output_stage_dir,
+    get_sample_name,
+    log_step_start,
+    sanity_check_inputs,
+    timer,
+    write_parameters_file,
+)
+
+
+@click.command(
+    "layout",
+    short_help="compute graph layouts that can be used to visualize components",
+    options_metavar="<options>",
+)
+@click.argument(
+    "pxl_file",
+    nargs=1,
+    required=True,
+    type=click.Path(exists=True),
+    metavar="PXL",
+)
+@click.option(
```

Comment:
I think boolean flags should always be False by default. Passing true or false explicitly to a flag always feels weird to me.
So I would rename this to --no-node-marker-counts or something like that.